packages/hash/frontend/src/pages/login.page/login-modal.tsx

```diff
@@ -9,28 +9,31 @@ import React, {
 import { useRouter } from "next/router";
 
 import { useMutation } from "@apollo/client";
-import { AuthModalLayout, AuthModalLayoutProps } from "./AuthModalLayout";
-import { LoginIntro as LoginIntroScreen } from "../../auth/login/LoginIntro";
-import { VerifyCode as VerifyCodeScreen } from "../../auth/VerifyCode";
+import { LoginIntro as LoginIntroScreen } from "./login-intro";
+import { VerifyCode as VerifyCodeScreen } from "../shared/verify-code";
```

Comment:
[optional] remove `as` (`LoginIntroScreen` + `./login-intro-screen`)? Not sure we benefit from on-the-fly renaming here. It's a bit confusing.

Reply:
I agree. I removed the `as *Screen` for now in [8c19e64](https://github.com/hashintel/hash/pull/468/commits/8c19e64f44905c778cdeca75b7c8d919b5e779b3). If it feels confusing, we can always add back plus there's a chance that flow might be refactored